src/components/DayPicker.jsx

```diff
@@ -129,6 +130,7 @@ const propTypes = forbidExtraProps({
 
 export const defaultProps = {
   // calendar presentation props
+  customRef: null,
```

Comment:
this seems an unnecessary property when you can wrap the component in a div and put a ref on that?